docs/topics_5/web-running-saved-analysis.adoc

```diff
@@ -0,0 +1,16 @@
+
+
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='running_saved_analysis_{context}']
+= Running a saved analysis
+
+You can run an analysis of the last saved version of a project from the *Analysis results* screen.
+
+.Procedure
+
+. In the {WebName}, click *Analysis results*.
+. Select a project.
+. Click *Run analysis*.
+
+The *Active analysis* progress bar displays the progress of your analysis.
```

Comment:
It's a bit repetitious. Suggest this: "A progress bar displays the progress of your analysis." I don't think you need the name of the bar or the fact that it is in the Active analysis section. That detail doesn't add much.